user/api.go

```diff
@@ -340,6 +341,7 @@ func (a *Api) GetStatus(res http.ResponseWriter, req *http.Request) {
 // @Produce  json
 // @Param role query string false "Role" Enums(clinic)
 // @Param id query string false "List of UserId separated by ,"
+// @Param emailVerified query boolean false "Filter users on emailVerified"
```

Comment:
Out of scope just giving more context to OpenApi doc.

```suggestion
// @Param role query string false "Role. Exactly one query parameter is required in role, id, emailVerified" Enums(clinic)
// @Param id query string false "List of UserId separated by ,. Exactly one query parameter is required in role, id, emailVerified"
// @Param emailVerified query boolean false "Filter users on emailVerified. Exactly one query parameter is required in role, id, emailVerified"
```